<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbConnectionDialog</class>
 <widget class="QWidget" name="DbConnectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Go to database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="0">
      <widget class="QLabel" name="spinBoxPort">
       <property name="text">
        <string>PDB</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="getDatabaseName"/>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lineEditLogin">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lineEditPassword">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="getPassword">
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="getHostName">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::StrongFocus</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="getPort">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>5432</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="getLogin"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lineEditHostName">
       <property name="text">
        <string>Host name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lineEditDatabaseName">
       <property name="text">
        <string>Database name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>getDatabaseName</tabstop>
  <tabstop>getHostName</tabstop>
  <tabstop>getLogin</tabstop>
  <tabstop>getPassword</tabstop>
  <tabstop>getPort</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
